docs/concepts/services-networking/service.md

```diff
@@ -403,6 +403,13 @@ allocate a port from a flag-configured range (default: 30000-32767), and each
 Node will proxy that port (the same port number on every Node) into your `Service`.
 That port will be reported in your `Service`'s `spec.ports[*].nodePort` field.
 
+If you want to specify particular IP(s) to proxy the port, you can set the `--nodeport-addresses` flag in kube-proxy to particular IP block(s) (which is supported since Kubernetes v1.10). The IP block(s) specifies the interface where the NodePort Services work. For example, if you start kube-proxy with flag `--nodeport-addresses=127.0.0.0/8`, kube-proxy will select only the loopback interface for NodePort Services. The `--nodeport-addresses` is defaulted to `0.0.0.0/0`, which means select all available interfaces and is in compliance with current NodePort behaviour. The following values are all valid for `--nodeport-addresses`:
+```
+default-route
```

Comment:
default-route is removed in the implementation.

Reply:
... The `--nodeport-addresses` is defaulted to `[]`, which means to select all ...

Sorry, there are some inconsistency between imp. and design document...

Reply:
OK. Updated.